docs/developer-docs/latest/setup-deployment-guides/configurations/required/middlewares.md

```diff
@@ -98,63 +117,180 @@ Strapi's core includes the following internal middlewares, mostly used for perfo
 - and [session](#session).
 
 ::: caution
-The following built-in middlewares are automatically added by Strapi: `errors`, `security`, `cors`, `query`, `body`, `public`, `favicon`. They should not be removed as it will throw an error.
+The following built-in middlewares are automatically added by Strapi: `errors`, `security`, `cors`, `query`, `body`, `public`, `session`, `favicon`. They should not be removed as it will throw an error.
```

Comment:
```suggestion
The following built-in middlewares are automatically added by Strapi: `errors`, `security`, `cors`, `query`, `body`, `public`, `session`, `favicon`. All of them except `session` should not be removed as it will throw an error.
```